graphql_query_derive/src/fragments.rs

```diff
@@ -13,7 +13,12 @@ impl GqlFragment {
     pub(crate) fn to_rust(&self, context: &QueryContext) -> Result<TokenStream, ::failure::Error> {
         let derives = context.response_derives();
         let name_ident = Ident::new(&self.name, Span::call_site());
-        let object = context.schema.objects.get(&self.on).expect("oh, noes");
+        let opt_object = context.schema.objects.get(&self.on);
+        let object = if let Some(object) = opt_object {
+            object
+        } else {
+            panic!("fragment '{}' cannot operate on unknown type '{}'", self.name, self.on);
+        };
```

Comment:
That's a lot better - I forgot we still had error messages as bad as the original one :/

Reply:
Well, it was easy to grep for to replace when I hit it :) .